src/themes/clean/templates/press/core/news/index.html

```diff
@@ -22,20 +22,41 @@ <h6>Posted by {{ item.posted_by.full_name }} on {{ item.posted|date_human }}</h6
             <h4>{% trans 'This journal currently has no news items to display' %}.</h4>
         {% endfor %}
     </div>
-    <div class="pagination-block">{{ page }}
-        <ul class="d-flex justify-content-center">
-            {% if news_items.has_previous %}
-                <a href="?page={{ news_items.previous_page_number }}" class="btn btn-outline-primary">&laquo;</a>
-                &nbsp;{% endif %}
-            {{ news_items.page.page_range }}
-            {% for page in news_items.paginator.page_range %}
-                <a href="?page={{ page }}"
-                   class="btn {% if news_items.number == page %}btn-primary{% else %}btn-outline-primary{% endif %}">{{ page }}</a>
-                &nbsp;
-            {% endfor %}
-            {% if news_items.has_next %}
-                <a href="?page={{ news_items.next_page_number }}" class="btn btn-outline-primary">&raquo;</a>
-            {% endif %}
-        </ul>
-    </div>
+    <section aria-label="pagination">
+        <div class="pagination-block">{{ page }}
```

Comment:
I think the `{{ page }}` here is a bug. You caught it in the other paste of this code and removed it. Maybe combine these two instances with a sub-template to make them easier to maintain?